src/app/shell/personal-cabinet/parent/children/children.component.ts

```diff
@@ -22,8 +22,8 @@ import { DeleteChildById, GetUsersChildren } from 'src/app/shared/store/parent.a
   styleUrls: ['./children.component.scss'],
 })
 export class ChildrenComponent extends ParentComponent implements OnInit, OnDestroy {
-  @Select(PaginatorState.childrensPerPage)
-  childrensPerPage$: Observable<number>;
+  @Select(PaginatorState.itemsPerPage)
+  itemsPerPage$: Observable<number>;
```

Comment:
revert this change please, because for children.component we should use the old pagination, there is not table here